soap/src/java/com/zimbra/soap/mail/type/Mountpoint.java

```diff
@@ -124,6 +134,7 @@ public void setReminderEnabled(Boolean reminderEnabled) {
         this.reminderEnabled = ZmBoolean.fromBool(reminderEnabled);
     }
 
+    @GraphQLQuery(name=GqlConstants.BROKEN, description="If \"tr\" is true in the request, broken is set if this is a broken link")
```

Comment:
`isBroken` ?

Reply:
value of BROKEN was is Broken, now changed var name also.